ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiIdentityProviderConfigService.scala

```diff
@@ -49,11 +49,13 @@ class ApiIdentityProviderConfigService(
         )
         jwksUrl <- requireJwksUrl(config.jwksUrl, "jwks_url")
         issuer <- requireNonEmptyString(config.issuer, "issuer")
+        audience <- optionalString(config.audience)(requireNonEmptyString(_, "audience"))
```

Comment:
The lingo in the .proto suggest that this field is optional, here however we are making it a required field. This will be 
- a backwards incompatible change
- forces admins to think up of a new name even if they'd rather go with the actual participant name here

Reply:
it's an optional field, otherwise it would not pass the test. but I agree the code is confusing here, i will rewrite

Reply:
I pushed the fix